### ISSUE TITLE
Blank dynamic spec for source far from phase centre

### ISSUE THREAD

**Issue by zhangxiang-planet** [cyriltasse/DynSpecMS#21]
When I use DynspecMS on a source ~10 deg away from the phase centre, I get blank (all elements are zero) dynamic spec. The W and W2 images are blank, too. The same happens to the OFF pointings.

A few notes:
1. The elevation of the source is relatively low (compared to previous observations where DynSpecMS worked perfectly). Phase centre at ~40 deg elevation, source at ~30 deg.
2. We observed with 64 channels per SB with NenuFAR. That's different from our usual setting (12 averaged channels, edge channels flagged).
3. No error message popped up during DynSpecMS processing.
4. I've made images with columns KMS_SUB and DDF_PREDICT. Both images look fine.

Here is the script. Any suggestions?
[dynspec.txt](https://github.com/cyriltasse/DynSpecMS/files/11882713/dynspec.txt)

**Comment by cyriltasse** [comment]
Weird - can you show the log file?

**Comment by zhangxiang-planet** [comment]
Sure. I didn't find a log file generated by DynSpecMS, so I re-ran the script and copied the screen output below:
[dynspecms_output.txt](https://github.com/cyriltasse/DynSpecMS/files/11883123/dynspecms_output.txt)

**Comment by zhangxiang-planet** [comment]
Hi Cyril. I think I've found the reason for the issue: I didn't set `uvrange` in my dynspecms script. 

I assumed the default would be no limit; however, the default limit was [1, 1000] in km, which is a bit high for NenuFAR... For this NenuFAR dataset, all the remote MAs were manually flagged, while all the core baselines got flagged by the uvrange limit. So there was no visibility left, and I got the empty dynamic spectra.

I am using `--uv=0.067,1000` and it works now.
